src/models/stocks.ts

```diff
@@ -0,0 +1,63 @@
+import axios from "axios";
+
+import { users } from "./user";
+import type { IStock } from "@interfaces";
+import { mongoose } from "@services";
+import { STOCK_API_URL } from "@src/types/values";
+
+class Stocks extends mongoose.Repository<IStock> {
+	private static readonly stockApiClient = axios.create({
+		baseURL: STOCK_API_URL
+	});
```

Comment:
we might as well just make a new client the way i do with webhooks in servoces/rest